pyramid/tweens.py

```diff
@@ -42,7 +42,7 @@ def excview_tween(request):
             provides = providedBy(exc)
             try:
                 response = _call_view(
-                    registry,
+                    request.registry,
```

Comment:
Hmm... Yes, I agree with you - this closure has not matter and do not prevents GC to resolve a cycle "registry-tween-registry" .
I could remove this change, if you think that it is absolutely useless.